app/components/forms/ErrorMessage.js

```diff
@@ -0,0 +1,11 @@
+import { StyleSheet } from "react-native";
+import { AppText } from "../AppText";
+
+export const ErrorMessage = ({ error, visible }) => {
+  if (!visible || !error) return null;
+  return <AppText style={styles.error}>{error}</AppText>;
+};
+
+const styles = StyleSheet.create({
+  error: { color: "red" },
```

Comment:
[nit pick] maybe move this to the default styles (`colors.red`?) and reference it as a hex?